app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>All Orders</h1>
+
+<% @orders.each do |order|%>
+  <tr>
+  <section id = "order_<%=order.id%>">
+      <td><p><%=link_to order.user.name, "/admin/profile/#{order.user.id}"%></p></td>
+      <td><p><%= "Order Number: #{order.id}" %></p></td>
+      <td><p><%= "Date Created: #{order.created_at.strftime("%m/%d/%Y")}" %></p></td>
+      <td><p><%= "Order Status: #{order.status}" %></p></td>
```

Comment:
Note: User Story didn't call for Order Status to be displayed, but given that we're hiding the button below based on status we made an exec decision to include it.